include.js

```diff
@@ -5967,6 +5967,14 @@ let cards = [
     author: 'Esinnation',
     githubLink: 'https://github.com/esinnation'
   }
+
+  {
```

Comment:
Hello @jxz3 , you seemed to have omitted a ',' around here.
As a result of this, the deployment preview won't display anything from the cards object.
![Screenshot (327)](https://user-images.githubusercontent.com/32850163/175084237-9adcd5b0-aaa2-4e19-a69f-a235cb0eeb23.png)

Reply:
Hi, I think I have fixed this now, please let me know if there is anything else :)

Reply:
Hello @jxz3 , thanks for fixing this.